src/main/kotlin/org/rucca/cheese/model/GetTaskParticipants200ResponseDataDTO.kt

```
@@ -0,0 +1,13 @@
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.v3.oas.annotations.media.Schema
+import javax.validation.Valid
+
+/** @param participants */
+data class GetTaskParticipants200ResponseDataDTO(
+    @field:Valid
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("participants", required = true)
+    val participants: kotlin.collections.List<TaskParticipantSummaryDTO>
+) {}
```

detekt

[warning] The class or object GetTaskParticipants200ResponseDataDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)